build_stage1.sh

```diff
@@ -39,7 +39,11 @@ cd ${BuildDir}
 cp -f ${GitDir}/rhel-atomic-rebuild.repo /etc/yum.repos.d/
 yum -y install ostree rpm-ostree glib2 docker libvirt epel-release libgsystem
 
-cp -f ${GitDir}/atomic7-testing.repo /etc/yum.repos.d/
+
+if [ -f ${GitDir}/atomic7-testing.repo ]; then
```

Comment:
Ok, I see.  So the repo is only enabled after we do an install.  Seems like it'd be worth a comment there.

Also, seems like this model requires that the host is destroyed and reinitialized each time?  Otherwise the default `yum` model will have the "higher version installed wins" semantics.  Fixing that would require changing the line above to something like:

`yum -y --disablerepo=atomic7-testing install ostree && yum -y --disablerepo=atomic7-testing distro-sync ostree` or something like that.